docs/explanation/security.md

```diff
@@ -15,3 +15,51 @@ Charmed Aether SD-Core enforces TLS encryption across all communication within t
 Each Charmed Aether SD-Core charm generates its private key and a certificate signing request (CSR). The CSR is then transmitted to a TLS certificate provider, which in turn signs the certificate and sends it back to the charm. Subsequently, the 5G network function utilizes this certificate to encrypt its communications with other network functions.
 
 By default, the TLS certificate provider employed is the [self-signed-certificates operator](https://charmhub.io/self-signed-certificates). However, users have the flexibility to utilize any TLS certificates provider as long as it supports the `tls-certificates` integration.
+
+## Access Control and Secure Communication in NMS
```

Comment:
This whole block is very verbose and can likely be reduced to maybe 3 sentences on authentication, authorization, and access